internal/api/schema.go

```diff
@@ -23,6 +23,8 @@ func schemaToProto(s schema.Schema) *stencilv1beta1.Schema {
 }
 
 func (a *API) CreateSchema(ctx context.Context, in *stencilv1beta1.CreateSchemaRequest) (*stencilv1beta1.CreateSchemaResponse, error) {
+	endFunc := a.newrelic.StartGenericSegment(ctx, "CreateSchema")
+	defer endFunc()
```

Comment:
Same here. rewrite these in one liner. 

https://github.com/goto/stencil/pull/10#discussion_r1433859809